content/k8s/kubernetes/k3s-quickstart/_index.md

```diff
@@ -0,0 +1,101 @@
+---
```

Comment:
@t-witchy I think you added this file?

It is missing some content. Before the Deployment, we install K3d and talk about other resources as well.

https://app.gitbook.com/o/2DzlYnPstQTFtiSgav55/s/j3rSnsmwqXaNyuH81o0A/practical-component-chapters/k8s-the-containers/activity-1/3.1.1-k3s-introduction